movie_library/utils/serializers/movie_actor_serializer.py

```diff
@@ -0,0 +1,24 @@
+from rest_framework import serializers
+from .actor_serializer import ActorSerializer
+# from .movie_serializer import MovieSerializer
+from ...models.movie_actor import MovieActor
+
+
+class MovieActorSerializer(serializers.ModelSerializer):
+    actors = serializers.SerializerMethodField("get_funct")
+    actors = ActorSerializer(actors, many=False)
```

Comment:
same as this 
https://github.com/DeyarIdhilie/backend_task/pull/7/files#r958536530